odoo80/Dockerfile

```diff
@@ -48,6 +46,8 @@ RUN pip install pyyaml && cd /tmp \
 RUN cd /tmp \
     && wget -q https://raw.githubusercontent.com/Vauxoo/odoo-network/8.0/addons/network/scripts/odoo-server/05-install-dependencies-python-v80.sh \
     && sh 05-install-dependencies-python-v80.sh
+RUN cd /tmp && wget -q https://raw.githubusercontent.com/vauxoo/odoo/8.0/requirements.txt \
+    && pip install -r requirements.txt
```

Comment:
No, when you put a version the package is upgraded or downgraded to match the version (if any available), it installed the required version instead of the latest as requested.